lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -243,25 +250,13 @@ export class TagsCreatorComponent implements OnInit, OnDestroy {
         return this._existingTagsPanelVisible;
     }
 
-    /**
-     * Hide input for typing name for new tag or for searching. When input is hidden then panel of existing tags is hidden as well.
-     */
-    hideNameInput(): void {
-        this.tagNameControlVisible = false;
-        this._existingTagsPanelVisible = false;
-        this.existingTagsPanelVisibilityChange.emit(this.existingTagsPanelVisible);
-        this.tagNameControlVisibleChange.emit(this.tagNameControlVisible);
```

Comment:
tagNameControlVisibleChange field seems unused now, should we remove it?

Reply:
It is getting used internally in **content-metadata/content-metadata.component.html** , as after removing this we are getting error.
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/106949107/8549585b-f542-4e50-b401-62897242550e)

Reply:
Yes, that error comes because here is two way binding used so if you remove tagNameControlVisibleChange event then you need remove also binding in html. 

So instead of 
[(tagNameControlVisible)] you would need have just [tagNameControlVisible] (so without () around). Then you should be able to remove tagNameControlVisibleChange.

You can also check tags creator component to see if that tagNameControlVisibleChange is really redundant or if we should call it in tags creator.

Reply:
Done